### PR TITLE
short-description : Fixes #383

### LINKED ISSUE #383: Limit the visible length of the probe descriptions
The probe descriptions can have arbitrary length.
We should hide them after a certain length (one line on reasonable window widths?).
We can consider making them expand on user actions.

### PR DESCRIPTION
**Pull request by shikhar-scs**
As mentioned by @georgf at #383, the description is now shortened and the length is adjustable.

Here's a preview

short

Please review.

### CODE REVIEW

File: new-pipeline/src/dashboards.js [hunk 1]
```diff
@@ -1077,7 +1077,14 @@ function getDescriptionWithLink(metric, channel, description) {
   var metricUrl = buildDictionaryURL(metric, channel, description);
   var descr = metric;
   if (description && (description.length > 0)) {
-    descr = description;
+	  
+	  if (description.length > 200) {
+		  descr = description.substring(0, 200);
+		  descr += ' ...'
```
Comment (georgf) [inline new-pipeline/src/dashboards.js:1083]: This has roughly the desired effect, but the better approach is to solve this using CSS.
We should probably use styling with `text-overflow: ellipsis;` etc. to achieve this effect.
https://developer.mozilla.org/en-US/docs/Web/CSS/text-overflow

Comment (shikhar-scs) [reply new-pipeline/src/dashboards.js:1083]: Um, ok :+1:

**Comment by shikhar-scs** [comment]
@georgf 
I tried the other approach. For something like `text-overflow: ellipsis;` to work, as mentioned on the link you provided me above, there needs to exist the following css properties in its parent div 

```
.overflow-div {
  overflow: hidden;
  white-space: nowrap;
}
```

Removing either of those makes the `ellipses` property redundant. But what I achieved from that is this

screenshot from 2018-01-31 08-28-22

The line gets cut abruptly, and `More Details` remains hidden

**Comment by georgf** [comment]
We could change how the elements are structured & styled to fix this, that is fine.
The pieces that affect this are:
- `getDescriptionWithLink()` in dashboards.js
- the element `#dist-caption` in dist.js & dist.html (and surrounding elements)
- the element `#measure-description` in evo.js & evo.html (and surrounding elements)
- the corresponding CSS

**Comment by georgf** [comment]
Hi Shikar, are you stuck with anything? Can i help?

**Comment by shikhar-scs** [comment]
Hey george !!! I've actually been a bit pre occupied with so much work . Thanks for asking though. I'll get back to work in a few hours 😅😅

**Comment by georgf** [comment]
Hi shikhar, would it be ok for someone else to take over this issue for now?
Happy to make it available to you again or to have you active on other bugs!

**Comment by shikhar-scs** [comment]
Yeah sure. Actually GSOC is keeping me a bit busy these days. With completely new projects, I have to invest a lot of time there. I'll be happy to be back once I get over with it. :sweat_smile: Sorry, for all the inconvenience. As of now, yeah anyone can take over this. No issues. :+1:

**Comment by georgf** [comment]
This was solved by #418 :)

**Pull request closed without merging**